### DISCUSSION TITLE
How to run "fsutil file createnew" on windows10 system?

### DISCUSSION THREAD

**Discussion by wood3n** [google/zx#351] [Help needed]
## Expected Behavior

run successfully

## Actual Behavior

run failed

## Steps to Reproduce the Problem

I want to use `fsutil file createnew` to create large size file on windows10 system, that's my code.

```js
#!/usr/bin/env zx

try {
  // 1GB
  const size = 1024 * 1024 * 1024;
  await $`fsutil file createnew bigfile.txt ${size}`;
} catch (p) {
  console.log(`Exit code: ${p.exitCode}`);
  console.log(`Error: ${p.stderr}`);
}
```

It will get such a problem

![image](https://user-images.githubusercontent.com/31716713/158614192-8ad3870d-b32f-492a-8a98-d0602e67464a.png)

## Specifications

- Version:  6.0.1
- Platform: windows 10 - 19044

**Comment by antongolub** [comment]
https://github.com/google/zx#maxbuffer
So you can override the limit:
```ts
$.maxBuffer = 1024 * 1024 * 1024
$`cmd which requires a very large buffer`
```

**Comment by wood3n** [comment]
> https://github.com/google/zx#maxbuffer So you can override the limit:
> 
> ```ts
> $.maxBuffer = 1024 * 1024 * 1024
> $`cmd which requires a very large buffer`
> ```

thank you, but it seems have no effect, still has the same problem.

```js
#!/usr/bin/env zx

try {
  const size = 1024 * 1024 * 1024;
  $.maxBuffer = size;
  // 1GB
  await $`fsutil file createnew bigfile.txt ${size}`;
} catch (p) {
  console.log(`Exit code: ${p.exitCode}`);
  console.log(`Error: ${p.stderr}`);
}
```
![image](https://user-images.githubusercontent.com/31716713/158619539-db9a5078-9c85-4306-b58e-c5b70342d733.png)

**Comment by antongolub** [comment]
Strict limit. I suggest adding a bit more bytes. 
Btw, does this util return a new file contets as stdout?

Perhaps the problem is with the quotes. Can you check plz:
```
fsutil file createnew bigfile.txt 1000
fsutil file createnew bigfile.txt "1000"

**Comment by wood3n** [comment]
@antongolub  I just test, it has nothing to do with quotes on windows.😅
